app/models/post.rb

```diff
@@ -2,4 +2,9 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] It will be great to create a method that returns the 5 most recent comments for a given post. This is a major requirement for this project. It seems you missed this requirement.
![Screenshot 2023-10-07 at 09 26 05](https://github.com/najibullahjafari/ruby_social_media/assets/32361963/a4efe6db-ae52-49f7-9524-f7e895f3a3f0)